packages/hydra-js/package.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "@volto-hydra/hydra-js",
+  "version": "1.0.0",
+  "description": "Provides a method to initialize a bridge which enables communication between Volt Hydra and your frontend",
+  "private": true,
+  "exports": {
+  ".": {
+    "import": "./hydra.js",
+    "require": "./hydra.js"
+  }
+},
```

@MAX-786 Formatting is off. Also, you only need separate `import` and `require` if you have a separate `.cjs` or `.mjs` file. If its the same file you, can just use a string
```suggestion
  ".": "./hydra.js"
},
```